src/node/rpc/frontend.h

```diff
@@ -109,8 +109,8 @@ namespace ccf
           }
         }
         ctx->set_error(
-          HTTP_STATUS_INTERNAL_SERVER_ERROR,
-          ccf::errors::InternalError,
+          HTTP_STATUS_SERVICE_UNAVAILABLE,
+          ccf::errors::PrimaryNotFound,
```

Comment:
Is it worth adding this to the changelog? And also checking that this is the return status we get from the ORC when the node is retiring?

Reply:
As discussed, the ORC client doesn't have to check the response and should keep retrying until it's seen itself as `TRUSTED` from the consensus.